src/App.js

```diff
@@ -77,23 +84,37 @@ class App extends Component {
           )}
           {/* This is not permanent; I just have no idea where else this is going. We can discuss the front end navigation
           layout in more detail in a future meeting. -JP */}
-          <div className="navbar-nav ml-auto">
+           <div className="navbar-nav ml-auto"> 
+
             <li className="nav-item">
               <Link to={"/accounts"} className="nav-link">
                 Accounts
               </Link>
             </li>
+            <li className="nav-item">
+              <Link to={"/loans"} className="nav-link">
+                Loans
+              </Link>
+            </li>
+            <li className="nav-item">
+              <Link to={"/profile"} className="nav-link">
+                Profile
+              </Link>
+            </li>
           </div>
         </nav>
 
         <div className="container mt-3">
           <Switch>
+            <Route exact path="/home" component={Home} />
```

Comment:
I don't think this is reachable

Reply:
Its reachable, you just have to test it out.